modules/identity-provider-provisioning-user-lookup-mapping.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-identity-provider.adoc
+
+[id="identity-provider-provisioning-user-lookup-mapping_{context}"]
+= Manually provisioning a user when using the lookup mapping method
+
+Typically, identities are automatically mapped to users during login. The `lookup` mapping method disables this automatic mapping, which requires you to provision users manually. If you are using the `lookup` mapping method, use the following procedure for each user after configuring the identity provider.
+
+.Procedure
+
+. Create an {product-title} user:
++
+[source,terminal]
+----
+$ oc create user <username>
+----
+
+. Create an {product-title} identity:
++
+[source,terminal]
+----
+$ oc create identity <identity_provider>:<identity_provider_user_id>
+----
++
+Where `<identity_provider_user_id>` is a name that uniquely represents the user in the identity provider.
+
+. Create a user/identity mapping for the created user and identity:
```

Comment:
The slash feels ambiguous to me. Does that mean a "user-to-identity" mapping?

Reply:
I would say so. This was pre-existing content, but I will try to see if there's a better way to state this in our guidelines

Reply:
Decided to just rewrite as "user identity mapping" since in the example it's named `useridentitymapping`